CHANGELOG/CHANGELOG-1.14.md

```diff
@@ -0,0 +1,88 @@
+- Install flows for nested Services in `EndpointDNAT` only when Antrea Multi-cluster is enabled. ([#5411](https://github.com/antrea-io/antrea/pull/5411), [@hongliangl])
+- Make rate-limit of features configurable to help with performance tuning. ([#5450](https://github.com/antrea-io/antrea/pull/5450), [@GraysonWu])
+- Change the default flow's action to `drop` in `ARPSpoofGuardTable` to effectively prevent ARP spoofing. ([#5378](https://github.com/antrea-io/antrea/pull/5378), [@hongliangl])
+- Update Go to v1.21. ([#5377](https://github.com/antrea-io/antrea/pull/5377), [@antoninbas])
+
+### Fixed
+
+- Remove NetworkPolicyStats dependency of MulticastGroup API to fix the empty list issue when users run `kubectl get multicastgroups` even when the Multicast is enabled. ([#5367](https://github.com/antrea-io/antrea/pull/5367), [@ceclinux])
+- Fix `antctl tf` CLI failure when the traceflow is using IPv6 address. ([#5588](https://github.com/antrea-io/antrea/pull/5588), [@Atish-iaf])
+- Fix a deadlock issue in NetworkPolicy Controller which causes a FQDN resolution failure. ([#5566](https://github.com/antrea-io/antrea/pull/5566) [#5583](https://github.com/antrea-io/antrea/pull/5583), [@Dyanngg] [@tnqn])
+- Fix NetworkPolicy span calculation to avoid out-dated data when multiple NetworkPolicies have the same selector. ([#5554](https://github.com/antrea-io/antrea/pull/5554), [@tnqn])
+- Use the first matching address when getting Node address to find the correct transport interface. ([#5529](https://github.com/antrea-io/antrea/pull/5529), [@xliuxu])
+- Fix rollback invocation after CmdAdd failure in CNI server and improve logging. ([#5548](https://github.com/antrea-io/antrea/pull/5548), [@antoninbas])
+- Add error logging when Antrea network's MTU exceeds Suricata's maximum supported value. ([#5408](https://github.com/antrea-io/antrea/pull/5408), [@hongliangl])
+- Do not delete IPv6 link-local route in route reconciler to fix cross-Node Pod traffic or Pod-to-external traffic. ([#5483](https://github.com/antrea-io/antrea/pull/5483), [@wenyingd])
+- Do not apply Egress to traffic destined for ServiceCIDRs to avoid performance issue and unexpected behaviors. ([#5495](https://github.com/antrea-io/antrea/pull/5495), [@tnqn])
+- Unify TCP and UDP DNS interception flows to fix invalid flow matching for DNS responses. ([#5392](https://github.com/antrea-io/antrea/pull/5392), [@GraysonWu])
+- Fix the burst setting of the `PacketInQueue` to reduce the DNS response delay when a Pod has any FQDN policy applied. ([#5456](https://github.com/antrea-io/antrea/pull/5456), [@tnqn])
+- Fix SSL library downloading failure in Install-OVS.ps1 on Windows. ([#5510](https://github.com/antrea-io/antrea/pull/5510), [@XinShuYang])
+- Do not attempt to join Windows agents to the memberlist cluster to avoid misleading error logs. ([#5434](https://github.com/antrea-io/antrea/pull/5434), [@tnqn])
```

Comment:
```suggestion
- Do not attempt to join Windows antrea-agents to the memberlist cluster to avoid misleading error logs. ([#5434](https://github.com/antrea-io/antrea/pull/5434), [@tnqn])
```

Reply:
Done